WEB_SERVICE/WEB/_MEDIUM/ARRAY_BASED_BATCH_QUERIES/meta.json

```diff
@@ -0,0 +1,53 @@
+{
+    "risk_rating": "medium",
+    "title": "Array-Based Batch Queries",
+    "short_description": "Exploiting a GraphQL server by sending array-based batch queries, potentially leading to Denial of Service attacks.",
+    "references": {
+      "Inigo": "https://inigo.io/blog/defeating_controls_with_array-based_query_batching"
+    },
+    "privacy_issue": false,
+    "security_issue": true,
+    "categories": {
+        "CWE_TOP_25": [
+        "CWE_400","CWE_20"
```

Comment:
I don't think `CWE-400` is part of `CWE-TOP-25`

Reply:
accurate! i will remove it, thanks.